german/the-black-hack.md

```diff
@@ -280,7 +281,7 @@ Beim Würfeln, um festzustellen ob sich Eigenschaften erhöhen, würfeln Krieger
 
 **Start-TP :** W6 + 4
 **TP pro Stufe/Erholung :** 1W6
-**Waffen & Rüstung :** Alle Schwerter, alle Bögen, Dolche, Wams, Lederrüstung, Kleine Schilde
+**Waffen & Rüstung :** Alle Schwerter, alle Bögen, Dolche, Wams, Lederrüstung, kleine Schilde
```

Comment:
Trifft es auch nicht richtig. Eigenbegriff? Regelbegriff?

Ich sage nur, wenn es in der Ausrüstungstabelle groß geschrieben ist, sollte es hier auch groß stehen. Auch, um daran zu erinnern, dass es hier nicht um eine frei formulierte Eigenschaft geht, sondern dass es regeltechnisch nur Große Schilde und Kleine Schilde gibt und jeder Schild in die eine oder die andere Kategorie gehört.